### DISCUSSION TITLE
Prettier changing imports in SCSS brakes build

### DISCUSSION THREAD

**Discussion by andriusign** [prettier/prettier#13777] [Q&A]
For some reason Prettier is changing `@import 'variables/constants'` into `@import url('variables/constants')` which is breaking my builds in Angular.

Tried with 2.5.1. and 2.7.1, but same error persists. Any idea if this was changed in any recent Prettier versions and how can this be disabled?

**Comment by fisker** [comment]
[Can't reproduce](https://prettier.io/playground/#N4Igxg9gdgLgprEAuEABAlgWwA4QE4wAEA5AG4CGe65ARgDZwDOA9JFIzObI8SADQgI2GOmiNkoSnggB3AAqUE4lOTozyAT3ECaecmADWcGAGVymOABl0UOMgBmqxnB17Dxk9n02A5shh4AK4uIHCYNHAAJpFRllw+geQ+cABi+JjkMCJQfiqBMBD8IAAWMJh0AOrF6PCMXmBwJko16KQ1GsjgjNogNs4Ecno+GQ5OIQBWjAAeJr4MAIqBEPCjdM4CXnj9nYxg3UXYVLAV6JEwxcgAHAAMG9LOFXrYnYdMcHikdgIAjkvwg0JlCByIwALS2KJRIp4OC-dAwwZJEZIRxrELOTDofxBdFzOAAQSyVBo+Tgcne1lsq3WIEYeMWyzsKLGAk4NBOZwuSAATKy9Og6L4AMIQTDI0KMACsRUCzgAKrRlKiaaRggBJKAxWAmMBUYT4zUmGAaBjUuAAX3NQA)

**Comment by ZeeshanMalik1** [comment]
this is actually a known behavior in Prettier where it tries to normalize CSS imports if it iss breaking your Angular build, it's likely because the compiler isn't expecting the url() function for local SCSS members the quickest way to fix this without fighting the formatter is to use a Prettier ignore comment right above the line that's giving you trouble.

try adding this in your **SCSS** file:
|....................................................................|
|   // prettier-ignore (this is a comment ) |
|@import 'variables/constants';                |
|.....................................................................|

alternatively, if this is happening everywhere, check if you have a .prettierrc file. Prettier shouldn't be forcing url() on simple strings unless it thinks the file is plain CSS. Make sure your file extension is strictly .scss and not .css.

If you're on a very old version of Angular, upgrading the sass-loader often resolves why the url() syntax breaks the build in the first place.
